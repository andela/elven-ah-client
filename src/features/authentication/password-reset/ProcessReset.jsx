import React from 'react';

const ProcessReset = () => (
  <div className="mx-auto centered col-xl-8 col-lg-8 col-md-6 col-sm-6 col-xs-6 my-5">
    <h2>Please wait ......</h2>
  </div>
);


export default ProcessReset;
