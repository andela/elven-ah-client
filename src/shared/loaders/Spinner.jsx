import React from 'react';
import spinner from '../assets/img/spinner.svg';

const Spinner = () => (
  <nav className="loader">
    <img src={spinner} alt="loading........" />
  </nav>
);

export default Spinner;
