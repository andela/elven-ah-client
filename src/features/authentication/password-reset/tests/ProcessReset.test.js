import React from 'react';
import { shallow } from 'enzyme';

import ProcessReset from '../ProcessReset';
import Spinner from '../../../../shared/loaders/Spinner';

describe('Test the password reset verification component', () => {
  it('renders without crashing', () => {
    const wrapper = shallow(<ProcessReset />);
    expect(wrapper.find('div'));
    expect(wrapper.find(Spinner));
    expect(wrapper).toMatchSnapshot();
  });
});
