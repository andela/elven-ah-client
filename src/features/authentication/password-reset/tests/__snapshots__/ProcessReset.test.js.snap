// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test the password reset verification component renders without crashing 1`] = `
<div
  className=""
>
  Please Wait...
</div>
`;
